python/neutron-understack/neutron_understack/neutron_understack_mech.py

```diff
@@ -230,6 +232,23 @@ def delete_port_precommit(self, context):
     def delete_port_postcommit(self, context):
         log_call("delete_port_postcommit", context)
 
+        network_id = context.current["network_id"]
+
+        if network_id == cfg.CONF.ml2_type_understack.provisioning_network:
+            connected_interface_uuid = self.fetch_connected_interface_uuid(
+                context.current
+            )
+            port_status = "Active"
```

Comment:
this variable can be inlined

Reply:
true, but in case this will change for something longer, i like to have it like that